architecture/additional_concepts/flannel.adoc

```diff
@@ -29,4 +31,9 @@ responsibile for:
 
 Each *flanneld* agent provides this infomation to a centralized *etcd* store so
 other agents on hosts can create an overlay network and route packets to
-any container within the *flannel* network.
\ No newline at end of file
+any container within the *flannel* network.
+
+The following diagram illustrates the architecture and data flow from one
+container to another using a *flannel* network:
+
+image::flannel.png[Flannel Communication]
```

Comment:
@sferich888 Any chance you could take a look at the diagram?

Also @danwinship I checked my OCP w/flannel lab I'm using to test. When I run an `ip addr`, I can confirm it lists docker0 and veth devices. Not sure about flannel0, but judging by the IP, it seems more like a descriptor for the network range rather than a device. Having said that, I'm okay with removing it.